components/services/tor/tor_launcher_impl.cc

```diff
@@ -87,7 +96,7 @@ void TorLauncherImpl::Launch(mojom::TorConfigPtr config,
 #if BUILDFLAG(IS_WIN)
   launchopts.start_hidden = true;
 #endif
-  launchopts.current_directory = config->binary_path.DirName();
+  launchopts.current_directory = args.GetProgram().DirName();
```

Comment:
Done.

Reply:
I'm having to restore this line, otherwise `BraveTorBrowserTest.ResetBridges` fails, and the following message is shown in the terminal:

```
Oct 24 14:45:44.000 [warn] Could not launch managed proxy executable at '../../apfggiafobakjahnkchiecbomjgigkkn/1.0.0/tor-snowflake-brave' ('No such file or directory').
```

Reply:
I've added a comment explaining why we need this.